packages/dd-trace/src/channel-itm.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+const dc = require('diagnostics_channel')
+module.exports = {
+  moduleLoadStart: dc.channel('dd-trace:moduleLoadStart')
```

Comment:
This should no longer be needed if each file now uses DC directly.

Reply:
Good catch, I forgot to delete it